metaseq/hub_utils.py

```diff
@@ -120,6 +140,7 @@ def load_model(self):
         task = tasks.setup_task(self.cfg.task)
 
         def _build_model(cfg, task):
+            cfg.model.distribute_checkpointed_activations = False
```

Comment:
Dumb q - is there a reason for this line to go here rather than `make_generation_fast_()`?

(I'm assuming it's a load ordering thing... asking cause wondering if there's some nice way to bunch all of these generation-specific override things together — for example, would it make sense to have this live as a default in ARG_OVERRIDES below, or something along those lines?

Reply:
Good point! This can be moved to INFERENCE_ARG_OVERRIDES. Turns out, I did not have to do this if I turn off sequence_parallel. So removing this for now.